screens/ProfileScreen.js

```diff
@@ -0,0 +1,129 @@
+import React, { useState } from "react";
+import {
+  Alert,
+  StyleSheet,
+  Text,
+  SafeAreaView,
+  View,
+  TextInput,
+  TouchableOpacity,
+  Image,
+  Button,
+} from "react-native";
+import { connect } from "react-redux";
+import firebase from 'firebase';
+
+const ProfileScreen = (props) => {
+    const [heightFeet, setHeightFeet] = useState('');
+    const [heightInch, setHeightInch] = useState('');
+    const [weight, setWeight] = useState('');
+    const [bmi, setBMI] = useState('');
+
+
+  const numbersOnly = (input) => {
+    let numbers = '1234567890';
```

Comment:
I'm sure this works, but you could've used typeof operator here instead
if **typeof**(input) === "number" => do something